Formula/osinfo-db-tools.rb

```diff
@@ -0,0 +1,27 @@
+class OsinfoDbTools < Formula
+  desc "Tools for managing the libosinfo database files"
+  homepage "https://libosinfo.org"
+  url "https://releases.pagure.org/libosinfo/osinfo-db-tools-1.1.0.tar.gz"
+  sha256 "a141cd2fc07c30d84801b5dbf6b11f2c2e708b0e81216277d052ac0b57fe546b"
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "libarchive"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --localstatedir=#{var}
+      --mandir=#{man}
+      --sysconfdir=#{etc}
```

Comment:
Do these need to be specified or is `--prefix=#{prefix}` sufficient?

Reply:
I've removed `--mandir`.
`--sysconfdir=#{etc}` could be needed for manual osinfo-db upgrade. It should have exactly the same value in `libosinfo` and `osinfo-db-tools`. Both packages have C routines that append `/osinfo` to the path to retrieve value from the database or to install a local database.

The `--sysconfdir` behavior is leveraged in osinfo-db package